cross-overlays.nix

```diff
@@ -9,7 +9,7 @@ self: super: {
 
   haskell = let inherit (super) haskell; in haskell // {
     packages = haskell.packages // {
-      ghcWasm = haskell.packages.ghc863.override (drv: {
+      ghcWasm = haskell.packages.ghc865.override (drv: {
```

Comment:
Should this be 865, or 881?

Reply:
This was later changed to 881 in the ghc bump PR